components/molecule/buttonGroup/src/index.js

```diff
@@ -119,9 +121,12 @@ MoleculeButtonGroup.propTypes = {
   onClick: PropTypes.func,
 
   /**
-   *  configure the gap between the buttons of the group
+   * configure the gap between the buttons of the group
    **/
-  spaced: PropTypes.oneOf(Object.values(SPACED))
+  spaced: PropTypes.oneOf(Object.values(SPACED)),
+
+  /** buttons should have a vertical layout */
+  vertical: PropTypes.bool
```

Comment:
being it a boolean prop, I'd suggest naming it `isVertical`... what do you think?